apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditBuyAmount/SelectBuyTokenFromListScreen/SelectBuyTokenFromListScreen.tsx

```diff
@@ -216,9 +216,9 @@ const Counter = ({counter}: {counter: number}) => {
   if (!isSearching) {
     return (
       <View style={styles.counter}>
-        <Text style={styles.counterText}>{strings.youHave}</Text>
+        <Text style={styles.counterTextBold}>{`${counter} ${strings.assets(counter)} `}</Text>
 
-        <Text style={styles.counterTextBold}>{` ${counter} ${strings.tokens(counter)}`}</Text>
+        <Text style={styles.counterText}>{strings.available}</Text>
```

Comment:
just to keep consistency
```suggestion
        <Text style={styles.available}>{strings.available}</Text>
```

Reply:
I'd say it's still counterText. It's inside the counter line:
```
      <View style={styles.counter}>
        <Text style={styles.counterTextBold}>{`${counter} ${strings.assets(counter)} `}</Text>

        <Text style={styles.counterText}>{strings.available}</Text>
      </View>
```

![image](https://github.com/Emurgo/yoroi/assets/1725956/54f2b3ba-42fa-413a-8a02-9e11d682f0d2)

A few lines down it's the same line but `assets found` instead of `assets available`. I wouldn't rename the style to each